src/App.js

```diff
@@ -5,6 +5,16 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './Components/PostList';
+
+const preparedPosts = [];
+
+posts.forEach((post) => {
+  const newPost = post;
+
+  newPost.user = users.find(user => user.id === post.userId);
+  preparedPosts.push(newPost);
+});
```

Comment:
```suggestion
const preparedPosts = posts.map((post) => ({
  ...post,
  user:
  comments:
}));
```
Находи тут сразу и комментарии, и пользователя